MatrixSDK/Data/MXRoomSummary.h

```diff
@@ -137,6 +138,16 @@ FOUNDATION_EXPORT NSString *const kMXRoomSummaryDidChangeNotification;
 
 #pragma mark - Data related to room state
 
+/**
+ The room type string value. Can be nil.
```

Comment:
I would add as provided by the homeserver in opposition to `roomType` that is computed locally.